### PR TITLE
Merge master to next

### PR DESCRIPTION
**Pull request by manuelblum**

### CODE REVIEW

**Review by nsams — APPROVED** [review]

**Review by nsams — CHANGES_REQUESTED** [review]
target branch must be next :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ All notable changes to this project will be documented in this file. This projec
     -   Allows maximum item-nesting of two levels
 -   Changes to MasterLayout
     -   The default values for content-spacing and header-height have changed slightly
+-   add onAfterSubmit to FinalForm
```
Comment (nsams) [inline CHANGELOG.md:59]: this is not an incomptiable change

Comment (manuelblum) [reply CHANGELOG.md]: yep that's true, should be a highlight :-)

**Comment by manuelblum** [comment]
fixed broken travis job

**Review by nsams — APPROVED** [review]

**Review by dkarnutsch — APPROVED** [review]

**Pull request merged**